code/modules/client/preferences/_preference.dm

```diff
@@ -107,6 +107,10 @@ GLOBAL_LIST_INIT(preference_entries_by_key, init_preference_entries_by_key())
 	/// will show the feature as selectable.
 	var/relevant_mutant_bodypart = null
 
+	/// If the selected species has this in its /datum/species/body_markings,
+	/// will show the feature as selectable.
+	var/relevant_body_markings = null
```

Comment:
These all should probably be unified into less copypasty preference constraints in another PR

Reply:
if all goes well, sooner rather than later